### ISSUE TITLE
remove `user-scalable=no`

### ISSUE THREAD

**Issue by JJC1138** [JJC1138/web#1]
Zooming is an accessibility feature so should be available (and `user-scalable=no` is ignored on iOS 10 for that reason).
